contracts/Account.cairo

```diff
@@ -204,9 +200,7 @@ end
 
 func hash_message{pedersen_ptr : HashBuiltin*}(message: Message*) -> (res: felt):
     alloc_locals
```

Comment:
I think this line is no longer needed, right?

Reply:
It's still needed here, otherwise `res_calldata` gets revoked in the hash chain.